CHANGELOG.md

```diff
@@ -433,6 +437,8 @@ more discussion).
 - `transform!` have been deprecated in favor of `transform_coordinates!`.
   ([#754][github-754])
 
+- `Ferrite.default_interpolation` has been deprecated in favor of `geometric_interpolation` [#953][github-953]
```

Comment:
I believe `default_interpolation` doesn't exist in the previous release(?) so if you want to mention it here perhaps mention that it has only been on master for quite some time but not in a release

Reply:
Its available in the latest released version. Are you perhaps thinking about `default_geometric_interpolation(::Interpolation)`?

Reply:
But only documented in devdocs: https://ferrite-fem.github.io/Ferrite.jl/stable/devdocs/elements/#Ferrite.default_interpolation-Tuple{Ferrite.AbstractCell}

Reply:
Hmm, why are these not the same function then? They do the same thing or?

Reply:
No, because `default_geometric_interpolation` gives the linear interpolation version of the given function interpolation, independently if the supplied function interpolation is higher order or linear. Used to get the default geometric interpolation for the FEValues (#695). 

With #806 we would use `geometric_interpolation` such that it always is correct, since passing the dofhandler gives access to the cell type. 

But I guess the default will in most cases work - if I recall correctly when we discussed this I think the argument was that it is more common to have a quadratic interpolation on a linear grid, than to use a quadratic grid.

Reply:
Yea okay, that make sense.